### ISSUE TITLE
long long not recognized

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#106]
```
long long _lastBytesNotification;
```

is translated into

```
var _lastBytesNotification: long,long?
```

**Comment by drkameleon** [comment]
Can you please verify this one?

I cannot reproduce it!

**Input:**
```objective-c
@implementation aClass

- (void)main {
    long long _lastBytesNotification;
}
@end
```

**Output:**
```swift
class aClass {

	func main() {
		var _lastBytesNotification: CLongLong
	
	}
	
}
```

**Comment by charlieMonroe** [comment]
Ok, here it works, but try this example:

```
@implementation aClass {
    long long _lastBytesNotification;
}

@end
```

**Comment by drkameleon** [comment]
Ahaaaa... OK. Now I got it. ;-)

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0.

----

**Example:**
```objective-c
@interface aClass 

@property (assign) long long a;

@end
    
@implementation aClass {
    long long b;
}

- (void)someFunc
{
    long long c;
}

@end
```

**Output:**
```swift
class aClass {

	var a: CLongLong?
	var b: CLongLong?
	
	func someFunc() {
		var c: CLongLong
	
	}
	
}
```
